libr/anal/p/anal_arm_cs.c

```diff
@@ -1895,10 +1895,8 @@ static int analop64_esil(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int l
 		break;
 	}
 	case ARM64_INS_CCMP:
-	case ARM64_INS_CCMN:
 	case ARM64_INS_TST: // cmp w8, 0xd
 	case ARM64_INS_CMP: // cmp w8, 0xd
-	case ARM64_INS_CMN: // cmp w8, 0xd
 		ARG64_APPEND(&op->esil, 1);
```

Comment:
Add a space before (

Reply:
i found some more cases like this on already existing code. so i merged and im fixing it in local .thanks for the contrib!